src/js_app.hpp

```diff
@@ -238,13 +252,9 @@ void AppClass<T>::constructor(ContextType ctx, ObjectType this_object, Arguments
     config.device_info.framework_name = framework_name;
     config.device_info.framework_version = framework_version;
 
-    auto realm_file_directory = default_realm_file_directory();
-    ensure_directory_exists_for_file(realm_file_directory);
-
-    SyncClientConfig client_config;
-    client_config.base_file_path = realm_file_directory;
-    client_config.metadata_mode = SyncManager::MetadataMode::NoEncryption;
-    client_config.user_agent_binding_info = get_user_agent();
+    // ensure_directory_exists_for_file(client_config.base_file_path);
+    util::try_make_dir(client_config.base_file_path);
```

Comment:
Should `ensure_directory_exists_for_file` be commented out? Is it more or less tested by `util::try_make_dir`?

Reply:
We don't need it - I'll remove it